tests/setup.py

```diff
@@ -13,7 +13,7 @@
     package_data={'': ['*.md']},
     include_package_data=True,
     install_requires=[
-        'ducktape@git+https://github.com/redpanda-data/ducktape.git@6e2af9173a79feb8661c4c7a5776080721710a43',
+        'ducktape@git+https://github.com/redpanda-data/ducktape.git@a60d8e93ac5f13554dae036465a28ece6e407df1',
```

Comment:
nit: In this commit message, it would be nice to know that the ducktape commit `a60d8e93ac5f13554dae036465a28ece6e407df1` is where ok_to_fail is introduced.

Reply:
i feel if someone were to navigate to the github commit, it would help explain it: https://github.com/redpanda-data/ducktape/commit/a60d8e93ac5f13554dae036465a28ece6e407df1